src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/catalog_price_rule.yml

```diff
@@ -0,0 +1,53 @@
+admin_catalog_price_rules_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Sell\Catalog\CatalogPriceRule:index'
+    _legacy_controller: AdminSpecificPriceRule
+    _legacy_link: AdminSpecificPriceRule
+
+admin_catalog_price_rules_search:
+  path: /
+  methods: [POST]
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Sell\Catalog\CatalogPriceRule:search'
+    _legacy_controller: AdminSpecificPriceRule
+    _legacy_link: AdminSpecificPriceRule:submitFilterspecific_price_rule
+
+##todo: implement create action #13716
```

Comment:
Let's not forget these `@todo` 😄

Reply:
It must be gone after this PR is rebased with create/edit PR for sure :smile: